JenkinsConsoleUtility/jcuSrc/Commands/GetAdoBuilds.cs

```diff
@@ -11,8 +11,10 @@ namespace JenkinsConsoleUtility.jcuSrc.Commands
 {
     class GetAdoBuilds : ICommand
     {
-        private const string buildsUrl = "https://dev.azure.com/PlayFabInternal/Main/_apis/build/builds?api-version=6.1-preview.6";
-        public static readonly List<string> filterIds = new List<string> { "refs/heads/master", "refs/heads/develop" };
+        private const string expectedFolderPrefix = "\\PlayFabSdk";
+        private const string pipelinesUrl = "https://dev.azure.com/PlayFabInternal/Main/_apis/pipelines";
+        private const string buildsUrl = "https://dev.azure.com/PlayFabInternal/Main/_apis/build/builds?definitions={definitionId}";
+        // public static readonly List<string> filterIds = new List<string> { "refs/heads/master", "refs/heads/develop" };
```

Comment:
not sure what you policy about commented code, consider removing if it was left accidentally

Reply:
It was not accidental, but it may be a required filter that is added at a later time.
At present it's not necessary, but it could become necessary... maybe in the next couple of months.
I'd like to leave it because those strings aren't inherently obvious and when I need it later, I don't want to have to look it up.